src/js/intlTelInput.js.ejs

```diff
@@ -429,7 +429,7 @@ Plugin.prototype = {
     var that = this;
 
     // check for cookie
-    var cookieAutoCountry = ($.cookie) ? $.cookie("itiAutoCountry") : "";
+    var cookieAutoCountry = (!!window.Cookies) ? Cookies.get("itiAutoCountry") : "";
```

Comment:
Ok thanks for the clarification. I actually disagree that this is more readable - I think it's obfuscated and confusing. Perhaps doing an explicit cast to boolean would be more readable. Either way, I think we should go for consistency for now. Also please can you run `grunt js` to build the JS files?